src/cart/crt_swim.c

```diff
@@ -819,7 +819,7 @@ static swim_id_t crt_swim_get_dping_target(struct swim_context *ctx)
 {
 	struct crt_grp_priv	*grp_priv = crt_gdata.cg_grp->gg_primary_grp;
 	struct crt_swim_membs	*csm = &grp_priv->gp_membs_swim;
-	struct crt_swim_target	 cst;
+	struct crt_swim_target   cst      = {0};
```

Comment:
is that a bug fix? do we rely on cst fields being 0ed?

Reply:
I can't see a code-path where this matters but the compiler thinks there is one, so I think this is a case of making the compiler happy.